packages/maskbook/src/plugins/PoolTogether/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { ChainId } from '@masknet/web3-shared'
+import { ONE_SECOND, ONE_WEEK_SECONDS } from './constants'
+import type { Pool } from './types'
+
+export const subtractDates = (dateA: Date, dateB: Date) => {
```

Comment:
plz use https://date-fns.org/v2.22.1/docs/sub replace the function.

Reply:
I rewrote this function, now it takes `seconds` and returns `{days, hours, minutes, seconds}`.

Reply:
why not use https://date-fns.org/v2.22.1/docs/intervalToDuration

Reply:
@iRhonin